aclk/agent_cloud_link.h

```diff
@@ -29,6 +29,15 @@
 #define ACLK_DEFAULT_PORT 9002
 #define ACLK_DEFAULT_HOST "localhost"
 
+#define ACLK_LWT_MSG    "{ \
```

Comment:
Why is this format string in a header file instead of in the sprintf call where people can read it?

Reply:
Good point not needed; will change

Reply:
hmmm now you cannot use sizeof and have to put constants for buffer size :/